src/main/java/com/github/onsdigital/babbage/api/endpoint/content/MaxAge.java

```diff
@@ -54,6 +62,40 @@ protected boolean verifyKey(String key) {
     }
 
     protected int getMaxAge(HttpServletRequest request) throws Exception {
+        if (appConfig().babbage().isLegacyCacheAPIEnabled()) {
+            CloseableHttpClient httpClient = HttpClients.createDefault();
+            RequestBuilder requestBuilder = RequestBuilder.create(request.getMethod())
+                .setUri(appConfig().babbage().getLegacyCacheProxyUrl());
```

Comment:
This would be slightly more readable if you stored `appConfig().babbage().getLegacyCacheProxyUrl()` in a constant. You can also do the same with `appConfig().babbage().isLegacyCacheAPIEnabled()`, as none of them will change, they can be saved as `private static final` in this class, for instance.